### PR TITLE
Replace uint with u32 in trpl/guessing-game.md

### PR DESCRIPTION
**Pull request by CarVac**
`uint` was recently deprecated, so in following the use of `i32` in the first parts, replace all copies of `uint` with `u32`.

### CODE REVIEW

File: src/doc/trpl/guessing-game.md [hunk 1]
```diff
@@ -239,7 +239,7 @@ use std::rand;
 fn main() {
     println!("Guess the number!");
 
-    let secret_number = (rand::random::<uint>() % 100u) + 1u;
+    let secret_number = (rand::random::<u32>() % 100u) + 1u;
```
Comment (csouth3) [inline src/doc/trpl/guessing-game.md]: This is currently breaking the build:
```
<anon>:10:50: 10:54 error: mismatched types: expected `u32`, found `usize` (expected u32, found usize)
<anon>:10     let secret_number = (rand::random::<u32>() % 100u) + 1u;
                                                           ^~~~
<anon>:10:58: 10:60 error: mismatched types: expected `u32`, found `usize` (expected u32, found usize)
<anon>:10     let secret_number = (rand::random::<u32>() % 100u) + 1u;
                                                                   ^~
```
because the `u` suffix means that those literals are of type `uint` still.  As it turns out, these suffixes can actually be entirely removed now due to integer fallback, e.g. this can become:
```rust
let secret_number = (rand::random::<u32>() % 100) + 1;
```
both here, and at the several other occurrences of this same line below here, and that will fix it. :)

Comment (steveklabnik) [inline src/doc/trpl/guessing-game.md]: Yes, that sounds like the plan.

Comment (CarVac) [inline src/doc/trpl/guessing-game.md]: Should I make another commit and issue a separate pull request?

I'm not fully sure of the procedure.

Comment (csouth3) [inline src/doc/trpl/guessing-game.md]: So the best way to go about this is to just make the additional changes to this file, and then:
```
git add guessing-game.md
git commit --amend
(do the commit)
git push -f origin master
```
That will update your PR with the new changes!
